bingart/cli.py

```
@@ -0,0 +1,335 @@
+    return cookie, False
+
+
+async def download_file(url, dest_path, session=None):
+    import urllib.request
+
+    logger = logging.getLogger("bingart")
+    loop = asyncio.get_running_loop()
+
+    try:
+        await loop.run_in_executor(
+            None, lambda: urllib.request.urlretrieve(url, str(dest_path))
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)